design-documents/service-isolation/caching-layer.md

```diff
@@ -0,0 +1,105 @@
+# Caching layer for Magento services
+
+## Overview
+
+Cache is one of the most important parts in a modern web application architecture.
+Magento caching layer should effectively improve:
+ * Performance
+ * Availability
+ * Scalability
+
+Caching should be implemented on several levels:
```

Comment:
I would also add here caching responses of the services (by reverse proxies).

Reply:
Added